Crawlers/twitter_accounts/adaderana.py

```diff
@@ -0,0 +1,91 @@
+import tweepy
+from elasticsearch import Elasticsearch, helpers
+from threading import Thread, Timer
+from datetime import datetime
+
+
+# Twitter API credentials
+consumer_key = "d4ziZbNtdOzBFnse8rux2iLpa"
+consumer_secret = "1104uapeYVVti40iLyehCms9nbBOkmihZJH8qujKBPRoO3E7G0"
+access_key = "858674738490724354-1BJJJXY3pMCzvo99PUO9600ppy2jsqI"
+access_secret = "f6wscyFUF0NbFNUwGn2DPHecrjIAHc4ru99LN6juQtZRB"
```

Comment:
Move these keys to a seperate file like an env file

Reply:
I have added these keys to a separate dotenv file.